e2e/process-services/widgets/text_widget.e2e.ts

```diff
@@ -0,0 +1,122 @@
+    let app = resources.Files.WIDGET_CHECK_APP.TEXT;
+    let deployedApp, process;
+
+    beforeAll(async (done) => {
+        let users = new UsersActions();
+
+        alfrescoJsApi = new AlfrescoApi({
+            provider: 'BPM',
+            hostBpm: TestConfig.adf.url
+        });
+
+        await alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+
+        processUserModel = await users.createTenantAndUser(alfrescoJsApi);
+
+        await alfrescoJsApi.login(processUserModel.email, processUserModel.password);
+        appModel = await appsActions.importPublishDeployApp(alfrescoJsApi, resources.Files.WIDGET_CHECK_APP.file_location);
+
+        let appDefinitions = await alfrescoJsApi.activiti.appsApi.getAppDefinitions();
+        deployedApp = appDefinitions.data.find((currentApp) => {
+            return currentApp.modelId === appModel.id;
+        });
+        process = await appsActions.startProcess(alfrescoJsApi, appModel, app.processName);
+        loginPage.loginToProcessServicesUsingUserModel(processUserModel);
+        done();
+    });
+
+    beforeEach(() => {
+        let urlToNavigateTo = `${TestConfig.adf.url}/activiti/apps/${deployedApp.id}/tasks/`;
+        browser.get(urlToNavigateTo);
+        taskPage.filtersPage().goToFilter(CONSTANTS.TASK_FILTERS.MY_TASKS);
+        taskPage.formFields().checkFormIsDisplayed();
+    });
+
+    afterAll(async (done) => {
+        await alfrescoJsApi.activiti.processApi.deleteProcessInstance(process.id);
+        await alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+        await alfrescoJsApi.activiti.adminTenantsApi.deleteTenant(processUserModel.tenantId);
+        done();
+    });
+
+    it('[C268157] General Properties', async () => {
+        let label = widget.textWidget().getFieldLabel(app.FIELD.simpleText);
+        expect(label).toBe('textSimple*');
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+        let placeHolder = widget.textWidget().getFieldPlaceHolder(app.FIELD.simpleText);
+        expect(placeHolder).toBe('Type something...');
+        widget.textWidget().setValue(app.FIELD.simpleText, 'TEST');
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeFalsy();
+    });
+
+    it('[C268170] Min-max length properties', async () => {
+        widget.textWidget().setValue(app.FIELD.textMinMax, 'A');
+        expect(widget.textWidget().getErrorMessage(app.FIELD.textMinMax)).toBe('Enter at least 4 characters');
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+        widget.textWidget().setValue(app.FIELD.textMinMax, 'AAAAAAAAAAA');
+        expect(widget.textWidget().getErrorMessage(app.FIELD.textMinMax)).toBe('Enter no more than 10 characters');
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+    });
+
+    it('[C268171] Input mask reversed checkbox properties', async () => {
```

Comment:
Could you change the name to : Should be able to set input mask reversed checkbox properties for Text widget ?